src/main/kotlin/com/cjbooms/fabrikt/model/KotlinGenModels.kt

```diff
@@ -67,7 +67,9 @@ private fun <T : GeneratedType> groupClasses(allModels: Collection<T>): Map<T, L
     val sealedClasses = allModels
```

Comment:
@ulrikandersen / @averabaq Do you think we should just remove this grouping? Grouping Sealed class implementations hasn't been a requirement since Kotlin 1.5

Reply:
Sorry for jumping late into this. By all means, my vote is an absolute "yes". There shouldn't be many adopters out there still on kotlin < 1.5, and if there are, they can still use Fabrikt up to this version. Still, not a big deal upgrading